docs/user/files.rst

```diff
@@ -9,21 +9,37 @@ This works even if it has been changed in the meantime.
 
     The available options might be limited by :ref:`privileges`.
 
+.. _download:
+
 Downloading translations
 ------------------------
 
 From the project or component dashboard, translatable files can be downloaded
-using the :guilabel:`Download original translation file` in the :guilabel:`Files`
-menu, producing a copy of the original file as it is stored in the upstream
-Version Control System.
+in the :guilabel:`Files` menu.
 
-You can also download the translation converted into one of widely used
-localization formats. The converted files will be enriched with data
-provided in Weblate such as additional context, comments or flags.
+The first option is to download original format as translated in the
+repository. In this case any pending changes in the translation are committed
+and up-to-date file is download without any conversion.
 
-Several file formats are available, including a compiled file
-to use in your choice of application (for example ``.mo`` files for GNU Gettext) using
-the :guilabel:`Files` menu.
+You can also download the translation converted into one of widely used
+localization formats. The converted files will be enriched with data provided
+in Weblate such as additional context, comments or flags. Several file formats
+are available via the :guilabel:`Customize download` in the :guilabel:`Files`
+menu:
+
+* gettext PO
+* XLIFF with gettext extensions
+* XLIFF 1.1
+* TermBase eXchange
```

Comment:
The actual list says "TBX" and "TMX" (in nb_NO anyway)

>gettext PO
XLIFF med gettext-endelser
XLIFF 1.1
**TBX
TMX**
gettext MO
CSV
Excel Open XML
JSON
Android String Resource
iOS-strenger

Reply:
It’s because nb_NO is a version behind this. You will see it for translations after the release